runners/katacoda/index.ts

```diff
@@ -240,6 +240,20 @@ export class Katacoda extends Runner {
 
     }
 
+
+    runBuildNg(step: Step, command: Command): RunResult {
+        let cdCommand = this.changeCurrentDir(path.join(this.getVariable(this.workspaceDirectory), command.parameters[0]));
+
+        this.steps.push({
+            "title": "Build the Angular Project",
+            "text": "step" + this.stepsCount + ".md"
+        });
+
+        this.renderTemplate("buildNg.md", this.outputPathTutorial + "step" + (this.stepsCount++) + ".md", { text: step.text, textAfter: step.textAfter, cdCommand: cdCommand, outputDir: command.parameters[1], useDevonCommand: this.getVariable(this.useDevonCommand) });
```

Comment:
Wouldn't this cause an error if no second parameter is specified for the build folder?

Reply:
If you look at the tutorial, i tested both cases and it worked as it should. If there is no value command.parameters[1] ten buildDir is null and in the markdown template it checks if the buildDir is not null.